invenio_rdm_records/ext.py

```diff
@@ -127,6 +127,15 @@ def init_config(self, app):
         if not app.config.get("COMMUNITIES_NAMESPACES"):
             app.config["COMMUNITIES_NAMESPACES"] = app.config["RDM_NAMESPACES"]
 
+        app.config.setdefault(
+            "RDM_FILES_DEFAULT_QUOTA_SIZE",
+            app.config.get("FILES_REST_DEFAULT_QUOTA_SIZE"),
+        )
+        app.config.setdefault(
+            "RDM_FILES_DEFAULT_MAX_FILE_SIZE",
+            app.config.get("FILES_REST_DEFAULT_MAX_FILE_SIZE"),
+        )
```

Comment:
Thinking about it, this has become quite awkward in terms of keeping backwards compatibility. Imagine the scenario where one has set `FILES_REST_DEFAULT_QUOTA_SIZE = 10GB`. I think we have to initially set `RDM_FILES_DEFAULT_QUOTA_SIZE = None` and only use it if has an actual value set, or otherwise default to `FILES_REST_DEFAULT_QUOTA_SIZE`. This can be done in the `get_files_quota` function.

We can also add a deprecation warning there, saying that "in the future either set `RDM_FILES_DEFAULT_QUOTA_SIZE`, or expect your quota to be 10**10`.